src/.prototype/musicutils/ts/musicutils.test.ts

```diff
@@ -137,8 +181,16 @@ describe('Music utilities', () => {
             expect(getPitchType('dha')).toBe('east indian solfege name');
         });
 
-        test("Generate pitch type of pitch aaa and expect 'uknown'", () => {
+        test("Generate pitch type of pitch aaa and expect 'unknown'", () => {
             expect(getPitchType('aaa')).toBe('unknown');
         });
+
+        test("Generate pitch type of pitch 3 and expect 'scalar mode number'", () => {
+            expect(getPitchType('3')).toBe('scalar mode number');
+        });
+
+        test("Generate pitch type of pitch n2.3 and expect 'scalar mode number'", () => {
```

Comment:
`n2.3` should produce `unknown`.

On line 490 in `musicutils.ts` it should be integer check is wrong. Use `isInteger()` instead.

Reply:
Yeah agreed.
Just to be sure "n2.0" should return `'scalar mode number'` right?

Reply:
I think generic note names only contain integers.

Reply:
Done.